src/client/Microsoft.Identity.Client/Extensibility/AcquireTokenOnBehalfOfParameterBuilderExtensions.cs

```diff
@@ -31,5 +32,26 @@ public static AcquireTokenOnBehalfOfParameterBuilder WithSearchInCacheForLongRun
             builder.Parameters.SearchInCacheForLongRunningObo = searchInCache;
             return builder;
         }
+
+        /// <summary>
+        /// Specifies an FMI path to be used for client assertion when exchanging MSI tokens.
```

Comment:
if this is a single pre-defined location then suggest switching to `the FMI path`

Reply:
Updated the docs.